api/external/nodes/manager/manager.proto

```diff
@@ -42,6 +56,13 @@ service NodeManagerService {
 			action: "infra:nodeManagers:get"
 		};
 	};
+
+	/*
+	Update a Node Manager
+
+	Update a node manager's metadata, such as its name, associated credential id, or data.
+	This is a PUT operation and it overwrites ALL of the existing node manager metadata. Include all fields, because a PUT operation overwrites any missing fields to empty ("").
```

Comment:
The last two sentences are kind of redundant, they could probably be combined into one.

Reply:
We try to be alarmingly clear about destructive actions.